docs/src/main/asciidoc/datastore.adoc

```diff
@@ -72,6 +72,9 @@ The following configuration options are available:
 | `spring.cloud.gcp.datastore.credentials.scopes` | https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Cloud GCP Cloud Datastore credentials | No | https://www.googleapis.com/auth/datastore
 | `spring.cloud.gcp.datastore.namespace` | The Cloud Datastore namespace to use | No | the Default namespace of Cloud Datastore in your GCP project
 | `spring.cloud.gcp.datastore.emulator-host` | The `hostname:port` of the https://cloud.google.com/datastore/docs/tools/datastore-emulator[Datastore Emulator] to connect to | No |
+| `spring.cloud.gcp.datastore.auto-emulator.enabled` | To enable the auto configuration to start a local instance of the Datastore Emulator. | No | `false`
```

Comment:
Renaming `remulator-host` to just `host` may cause compatibilities issues. How to handle this? This exactly why I named the new emulator properties as `auto-emulator` haha

Reply:
What kind of incompatibilities? You mean backwards incompatibility? You can still leave `remulator-host` and mark it as deprecated.

Reply:
But I will manually set the new configuration with the `emulator-host` if it is valid? We need to keep compatibility to those who already uses the `emulator-host`.

And will be weird since other modules like pubsub already uses `emulator-host`. Should we create an issue to fix them and continue with this new approach?